unlock-app/src/components/content/VerificationContent.tsx

```diff
@@ -35,7 +35,14 @@ export const VerificationContent = ({
         <title>{pageTitle('Verification')}</title>
       </Head>
       {account && <Account network={network} account={account} />}
-      <VerificationStatus data={data} sig={sig} hexData={hexData} />
+      {account && (
+        <VerificationStatus
+          data={data}
+          sig={sig}
+          hexData={hexData}
+          account={account}
```

Comment:
We now pass the account as it is used to detremine of we should load the metadata